lib/ronin/fuzzing/core_ext/string.rb

```diff
@@ -138,7 +138,8 @@ def repeating(lengths,&block)
   # Incrementally fuzzes the String.
   #
   # @param [Hash{Regexp,String,Symbol => Enumerable,Symbol}] substitutions
-  #   Patterns and their substitutions.
+  #   Patterns and their substitutions. Symbols are resolved to
+  #   [Ronin::Support::Text::Patterns](https://ronin-rb.dev/docs/ronin-support/Ronin/Support/Text/Patterns.html).
```

Comment:
Missing the word "constants". Maybe "Symbols are resolved to constants in ..." or "resolved to Ronin::Support::Text::Patterns constants".